tests/integration/devfile/cmd_devfile_describe_test.go

```diff
@@ -0,0 +1,104 @@
+package devfile
+
+import (
+	"github.com/openshift/odo/pkg/component"
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/odo/tests/helper"
+)
+
+var _ = Describe("odo devfile describe command tests", func() {
+	var namespace, context, currentWorkingDirectory, originalKubeconfig string
+
+	// Using program command according to cliRunner in devfile
+	cliRunner := helper.GetCliRunner()
+
+	// This is run before every Spec (It)
+	var _ = BeforeEach(func() {
+		if os.Getenv("KUBERNETES") != "true" {
+			Skip("Plain Kubernetes scenario only, skipping")
+		}
+		SetDefaultEventuallyTimeout(10 * time.Minute)
+		context = helper.CreateNewContext()
+		os.Setenv("GLOBALODOCONFIG", filepath.Join(context, "config.yaml"))
+		helper.CmdShouldPass("odo", "preference", "set", "Experimental", "true")
+		originalKubeconfig = os.Getenv("KUBECONFIG")
+		helper.LocalKubeconfigSet(context)
+		namespace = cliRunner.CreateRandNamespaceProject()
+		currentWorkingDirectory = helper.Getwd()
+		helper.Chdir(context)
```

Comment:
Let's not change the directory

Reply:
I merely ported the existing test to devfile so sure, I could do that but then why was it allowed for the existing test?

Reply:
We currently have a issue to remove those https://github.com/openshift/odo/issues/3801.